src/betagouv.ts

```diff
@@ -162,21 +171,23 @@ const betaOVH = {
     const url = `/email/domain/${config.domain}/account`;
 
     try {
-      return await ovh.requestPromised('GET', url, {});
+      return await ovh.requestPromised("GET", url, {});
     } catch (err) {
-      if (err.error === 404) return null;
+      if (err.error == "404") return null;
       throw new Error(`OVH Error GET on ${url} : ${JSON.stringify(err)}`);
     }
   },
   changePassword: async (id, password) => {
     const url = `/email/domain/${config.domain}/account/${id}/changePassword`;
 
     try {
-      await ovh.requestPromised('POST', url, { password });
+      await ovh.requestPromised("POST", url, { password });
     } catch (err) {
       throw new Error(`OVH Error on ${url} : ${JSON.stringify(err)}`);
     }
-  },
+  }
 };
 
-module.exports = { ...betaGouv, ...betaOVH };
+export {
+  betaGouv, betaOVH
+};
```

Comment:
ne devrait-ce pas etre :

```suggestion
export {
  ...betaGouv, ...betaOVH
};
```

?

Reply:
OK, non je comprend, t'as scindé les deux volontairements, car ça a plus de sens de distinguer les fonctions qui appelle ovh, de celle qui utilise l'api betagouv.

Reply:
Nop tu peux pas deconstruire dans un export, j'ai du modifier en conséquence. L'autre solution c'est d'exporter toutes les fonctions une par une.

Reply:
ok, mais un truc du style 
```
const betagouvLib = {..betaGouv, ...betaOVH}
export default betagouvLib
# ou
export betagouvLib
```

marche non ?

Reply:
Bonne remarque, mais en fait ce que je me demande c'est pourquoi c'est divisé comme ça ? Je vois juste un wrapper autour de github (pourquoi c'est pas avec les autres ?) et un autour de OVH. Du coup je trouvait ça plus logique de bien diviser les 2. Mais je peux tout regrouper sans problèmes

Reply:
je suis d'accord avec toi, moi ça me va comme ça, ça me semble plus clair